Dockerfile

```diff
@@ -48,15 +48,15 @@ RUN echo "@edge https://nl.alpinelinux.org/alpine/edge/main" >> /etc/apk/reposit
  && rm libiconv.tar.gz \
  && cd /tmp/src/libiconv-$LIBICONV_VERSION \
  && ./configure --prefix=/usr/local \
- && make \
+ && make -j$(getconf _NPROCESSORS_ONLN)\
  && make install \
  && libtool --finish /usr/local/lib \
  && cd /mastodon \
  && rm -rf /tmp/* /var/cache/apk/*
 
 COPY Gemfile Gemfile.lock package.json yarn.lock /mastodon/
 
-RUN bundle config build.nokogiri --with-iconv-lib=/usr/local/lib --with-iconv-include=/usr/local/include \
+RUN bundle config build.nokogiri --with-iconv-lib=/usr/local/lib --with-iconv-include=/usr/local/include --jobs $(getconf _NPROCESSORS_ONLN)\
```

Comment:
I'm not a Ruby / Bundler expert, but according to this article: https://robots.thoughtbot.com/parallel-gem-installing-using-bundler the ```--jobs``` option can be applied to every bundler config command. Worked for me.